src/Compilers/CSharp/Portable/Binder/RefSafetyAnalysis.cs

```diff
@@ -775,6 +776,58 @@ private void VisitObjectCreationExpressionBase(BoundObjectCreationExpressionBase
                         node.ArgsToParamsOpt,
                         _localScopeDepth,
                         _diagnostics);
+
+                    if (node.InitializerExpressionOpt is { })
+                    {
+                        // Object initializers are different than a normal constuctor in that the 
```

Comment:
>constuctor [](http://example.com/codeflow?start=75&length=10)

Typo: "constructor"

Reply:
You've only found one typo? There have to be more.